furms-model-api/src/main/java/io/imunity/furms/api/users/UserService.java

```diff
@@ -12,6 +12,7 @@
 public interface UserService {
 	List<User> getAllUsers();
 	List<User> getFenixAdmins();
+	void invite(String email);
```

Comment:
This name is too broad, this should be inviteFenixAdmin - under the hood invite for become fenix, site or other admin will look differently.

Reply:
i will fix this in next commit.